modules/ROOT/pages/upgrade-custom-policy-mule3.adoc

```diff
@@ -0,0 +1,35 @@
+= Upgrade Custom Policies for Mule 3
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+In API Manager, you can upgrade the version of your custom policies for your Mule 3 applications.
+
+== Uploading a New Version of Your Custom Policy
+
+. In API Manager, go to *Custom Policies*, and click *Add Custom Policy*.
+. Select the *Policy for Runtimes olden than Mule 4*.
+. In the *Name* field, type the name of your new policy version. For example, `Regex Filter Policy v2`.
+. Upload your YAML and XML files for your custom policy's new version.
++
+image::upgrade-custom-policy-mule3-08395.png[align=center]
+. Click *Add*.
+
+== Upgrading your Custom Policy Version in your API Instance
+
+. In API Manager, go to *API Administration*, and select the version of the API instance you want to upgrade its custom policy.
+. Click *Policies*.
+. Expand the policy name you want to upgrade, click *Actions*, and select *Change Version*.
++
+image::upgrade-custom-policy-mule3-3e0bb.png[align=center]
+. Select the new version of your custom policy, and click *Configure Policy*.
```

Comment:
We should explain that in the list of the modal that opens, will list all custom policies.

So expect to see 
- Regex Filter Policy
- Regex Filter Policy v2
- Other Policy
- Other Policy-2
- Other Policy-new

Reply:
A popup will show an all the custom policies of tour business group will be listed (check this [Sergio])

Reply:
Same rules applies if a policy in the list has unmet requirements.